### ISSUE TITLE
Ability to disable preinstalled DevHub

### ISSUE THREAD

**Issue by maackle** [holochain/launcher-tauri#91]
**Describe the solution you'd like**

DevHub should be able to be disabled like any other app. It makes sense that it would not be able to be uninstalled, but disabling should be possible.

**Describe alternatives you've considered**

I can't think of any alternative

**Comment by matthme** [comment]
I agree. The reason I removed the option to disable it for now is that a bug in holochain (at least in v0.0.162 and v0.0.165) seemed to break apps when they were disabled once and I didn't want that to happen with the devhub. It would a) break the app library and b) force you to reinstall the devhub (which currently is only possible through a factory reset of the launcher as a whole) which then would change your developer pubkey, so you wouldn't be able to make changes or create new releases to an app you uploaded to the devhub earlier.
The right thing to do once the mentioned bug is fixed is probably to allow disabling the devhub but show a warning that the app library depends on it.

**Comment by Connoropolous** [comment]
> "(which currently is only possible through a factory reset of the launcher as a whole) which then would change your developer pubkey, so you wouldn't be able to make changes or create new releases to an app you uploaded to the devhub earlier."

@matthme this situation is avoidable :) we've taken steps with Acorn to keep keys intact while migrating to new versions. 
1. make it so that the folder structure of conductor files and keys are that they sit beside each other as siblings, not conductor->keys 
2. setup namespaces for both: e.g. conductor-v1, keystore-v1
3. when performing factory resets, just delete the contents of conductor-v1. 
4. when delivering breaking changes to the keystore in a new version (rare!), bump the keystore-vX. Otherwise, only bump the `conductor-vX` path for the version 
👍

**Comment by matthme** [comment]
Thanks for making that connection! Versioning of files seems like a good idea, I think Guillem actually implemented this already with the most recent upate of lair-keystore and we have to do it for conductors anyway since we are running multiple versions of Holochain alongside each other. The problem in this case I think is that removing keys is actually intended behavior of a factory reset but there is no way to get the DevHub running again *other* than through a factory reset.
